docs/workers.md

```diff
@@ -225,7 +225,6 @@ expressions:
     ^/_matrix/client/(api/v1|r0|unstable)/joined_groups$
     ^/_matrix/client/(api/v1|r0|unstable)/publicised_groups$
     ^/_matrix/client/(api/v1|r0|unstable)/publicised_groups/
-    ^/_synapse/client/password_reset/email/submit_token$
```

Comment:
tangential, but we may as well fix it while we're here. 

This line was added in #8227. AFAICT, it has never worked on a worker.